Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3-alpine
+MAINTAINER Tecnativa <info@tecnativa.com>
+ENTRYPOINT ["/usr/src/app/entrypoint.sh"]
+ENV GIT_AUTHOR_NAME=git-aggregator \
+    EMAIL=https://hub.docker.com/r/tecnativa/git-aggregator
+# HACK Install git >= 2.11, to have --shallow-since
+# TODO Remove HACK when python:alpine is alpine >= v3.5
+RUN apk add --no-cache --repository http://dl-cdn.alpinelinux.org/alpine/v3.5/main git
```

Comment:
@Yajo Hrmmmm so remember when I reported that this didn't work in the Odoo Docker container? Well it works here.

I took it a step further and created a dockerfile with just this in it, and it still worked. My curiosity is peaked, so I'll probably fool around with this a bit more in that context.